diff -Naur fox-1.6.57/configure fox-1.6.57-patch1/configure
--- fox-1.6.57/configure	2019-01-23 16:20:09 +0000
+++ fox-1.6.57-patch1/configure	2019-01-23 17:03:20 +0000
@@ -704,6 +704,7 @@
 LT_REVISION
 LT_CURRENT
 LT_RELEASE
+FOX_LIBS
 FOX_PATCH_LEVEL
 FOX_MINOR_VERSION
 FOX_MAJOR_VERSION
@@ -3254,6 +3255,9 @@
 FOX_MINOR_VERSION=6
 FOX_PATCH_LEVEL=57

+FOX_LIBS="-L$libdir -lFOX-${FOX_MAJOR_VERSION}.${FOX_MINOR_VERSION}"
+
+
 # Binaries compiled with this patchlevel of FOX will need at least
 # version MAJOR.MINOR.INTERFACE_VERSION to run. The assertion
 # FOX_INTERFACE_VERSION >= FOX_BINARY_VERSION must hold.
diff -Naur fox-1.6.57/configure.ac fox-1.6.57-patch1/configure.ac
--- fox-1.6.57/configure.ac	2019-01-23 16:20:42 +0000
+++ fox-1.6.57-patch1/configure.ac	2019-01-23 16:12:29 +0000
@@ -14,6 +14,9 @@
 FOX_MINOR_VERSION=fox_minor
 FOX_PATCH_LEVEL=fox_level
 
+FOX_LIBS="-L$libdir -lFOX-${FOX_MAJOR_VERSION}.${FOX_MINOR_VERSION}"
+AC_SUBST(FOX_LIBS)
+
 # Binaries compiled with this patchlevel of FOX will need at least
 # version MAJOR.MINOR.INTERFACE_VERSION to run. The assertion
 # FOX_INTERFACE_VERSION >= FOX_BINARY_VERSION must hold.
diff -Naur fox-1.6.57/fox.pc.in fox-1.6.57-patch1/fox.pc.in
--- fox-1.6.57/fox.pc.in	2019-01-23 16:20:21 +0000
+++ fox-1.6.57-patch1/fox.pc.in	2019-01-23 16:12:29 +0000
@@ -7,7 +7,7 @@
 X_BASE_LIBS="@X_BASE_LIBS@"
 X_EXTRA_LIBS="@X_EXTRA_LIBS@"
 GL_LIBS="@GL_LIBS@"
-FOX_LIBS=-lFOX-@FOX_MAJOR_VERSION@.@FOX_MINOR_VERSION@
+FOX_LIBS=@FOX_LIBS@
 
 Name: FOX
 Description: The FOX Toolkit
